examples/ui.md

```diff
@@ -491,6 +491,21 @@ You can also get an alert with more padding and large icon by specifying `large`
 <x-icon name="chevron-down" size="xs" class="md:h-3 md:w-2" />
 ```
 
+### Chevron Down/Up component
```

Comment:
🤔 "open/close", "Down/Up", what do you thing to uniform those actions in "toggle"?
`<x-ark-chevron-toggle />`

Reply:
updated, thanks :+1: